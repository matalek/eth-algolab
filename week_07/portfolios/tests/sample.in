1 2
2000 160
10000
1000 80 2500
1000 80 2499
3 7
2000 160
2000 160
2000 60
10000 0 0
0 10000 0
0 0 400
1000 80 2500
1000 80 1250
1000 80 1249
1000 60 466
1000 60 465
1000 30 64
1000 30 63
0 0