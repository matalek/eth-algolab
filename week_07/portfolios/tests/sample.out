Yes.
No.
Yes.
Yes.
No.
Yes.
No.
Yes.
No.
